doc/man3/EVP_MD_meth_new.pod

```diff
@@ -84,7 +84,15 @@ together.  The available flags are:
 
 =item EVP_MD_FLAG_ONESHOT
 
-This digest method can only handles one block of input.
+This digest method can only handle one block of input.
+
+=item EVP_MD_FLAG_XOF
+
+This digest method is an extensible-output function (XOF).
```

Comment:
Most importantly this signals that the digest supports the EVP_MD_CTRL_XOF_LEN control.